column.go

```diff
@@ -370,6 +365,10 @@ func (cl *columnLoader) open(file *File, path []string) (*Column, error) {
 		}
 	}
 
+	c.fields = make([]Field, len(c.columns))
+	for i, column := range c.columns {
+		c.fields[i] = column
+	}
```

Comment:
Perhaps we can use `copy`?
```suggestion
        copy(c.fields, c.column)
```

Reply:
Ah nvm, the type does not match.